### DISCUSSION TITLE
Welcome to Automatiko Discussions!

### DISCUSSION THREAD

**Discussion by mswiderski** [automatiko-io/automatiko-engine#48] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arskahtech** [comment]
Hi Maciej, great work! 

I have a question: do you have any python based functions workflow example?

Thanks!

**Reply by mswiderski** [reply]
> Hi Maciej, great work!

hi, thanks!

> I have a question: do you have any python based functions workflow example?

can you be more specific? Are you looking for python based workflow engine or how you could consume python service from workflow?

**Comment by arskahtech** [comment]
I am looking for a workflow that is made up of python written functions

Thanks

**Reply by mswiderski** [reply]
I believe we could achieve something like this by utilising GraalVM capabilities to integrate both java and python. If you're interested in take that further please start new discussion thread (just noticed this is in the welcome one ;))
